webapp/app/js/perf.js

```diff
@@ -82,6 +82,14 @@ perf.factory('PhSeries', ['$http', 'thServiceDomain', function($http, thServiceD
           });
 
         if (seriesSummary) {
```

Comment:
The way this code is written, it kinda implies that we found a seriesSummary even if we didn't. Let's rename seriesSummary to just series.